homeassistant/components/weatherflow/sensor.py

```diff
@@ -0,0 +1,409 @@
+        key="rain_rate",
+        translation_key="rain_rate",
+        state_class=SensorStateClass.MEASUREMENT,
+        device_class=SensorDeviceClass.PRECIPITATION_INTENSITY,
+        icon="mdi:weather-rainy",
+        native_unit_of_measurement=UnitOfVolumetricFlux.MILLIMETERS_PER_HOUR,
+    ),
+    WeatherFlowSensorEntityDescription(
+        key="relative_humidity",
+        translation_key="humidity",
+        native_unit_of_measurement=PERCENTAGE,
+        device_class=SensorDeviceClass.HUMIDITY,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    WeatherFlowSensorEntityDescription(
+        key="rssi",
+        native_unit_of_measurement=SIGNAL_STRENGTH_DECIBELS_MILLIWATT,
+        device_class=SensorDeviceClass.SIGNAL_STRENGTH,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        state_class=SensorStateClass.MEASUREMENT,
+        entity_registry_enabled_default=False,
+        event_subscriptions=[EVENT_STATUS_UPDATE],
+    ),
+    WeatherFlowSensorEntityDescription(
+        key="station_pressure",
+        translation_key="station_pressure",
+        native_unit_of_measurement=UnitOfPressure.MBAR,
+        device_class=SensorDeviceClass.PRESSURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=5,
+        imperial_suggested_unit=UnitOfPressure.INHG,
+    ),
+    WeatherFlowSensorEntityDescription(
+        key="solar_radiation",
+        native_unit_of_measurement=UnitOfIrradiance.WATTS_PER_SQUARE_METER,
+        device_class=SensorDeviceClass.IRRADIANCE,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    WeatherFlowSensorEntityDescription(
+        key="up_since",
+        translation_key="up_since",
+        device_class=SensorDeviceClass.TIMESTAMP,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        entity_registry_enabled_default=False,
+        event_subscriptions=[EVENT_STATUS_UPDATE],
+    ),
+    WeatherFlowSensorEntityDescription(
+        key="uv",
+        translation_key="uv",
+        native_unit_of_measurement=UV_INDEX,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    WeatherFlowSensorEntityDescription(
+        key="vapor_pressure",
+        translation_key="vapor_pressure",
+        native_unit_of_measurement=UnitOfPressure.MBAR,
+        device_class=SensorDeviceClass.PRESSURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        imperial_suggested_unit=UnitOfPressure.INHG,
+        suggested_display_precision=5,
+    ),
+    ## Wind Sensors
+    WeatherFlowWindSensorEntityDescription(
+        key="wind_gust",
+        translation_key="wind_gust",
+        device_class=SensorDeviceClass.WIND_SPEED,
+    ),
+    WeatherFlowWindSensorEntityDescription(
+        key="wind_lull",
+        translation_key="wind_lull",
+        device_class=SensorDeviceClass.WIND_SPEED,
+    ),
+    WeatherFlowWindSensorEntityDescription(
+        key="wind_speed",
+        translation_key="wind_speed",
```

Comment:
Device class translations

Reply:
I think here we need to keep the average ... because wind_speed is already covered by the device class right?